modules/kmm-building-in-cluster.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-building-in-cluster_{context}"]
+
+= Building in the cluster
+
+KMM can build module loader images in the cluster. Follow these guidelines:
+
+* Provide build instructions using the `build` section of a kernel mapping.
+* Copy the `Dockerfile` for your container image into a `ConfigMap` resource, under the `dockerfile` key.
+* Ensure that the `ConfigMap` is located in the same namespace as the `Module`.
+
+KMM checks if the image name specified in the `containerImage` field exists.
+If it does, the build is skipped.
+
+Otherwise, KMM creates a https://docs.openshift.com/container-platform/4.12/cicd/builds/build-configuration.html[`Build`] resource to build your image.
```

Comment:
This should be an xref (see the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references) on this). Since xrefs can't go in modules, you could add it as an additional resource in the assembly underneath where this module is included.

Reply:
Moved link to the assembly.